We can win the race

LionAlar opened the pull request:
Созданы 3 класса:
Car - автомобиль.

Поля - название и скорость
Методы - геттеры и сеттеры полей, проверка скорости на корректность и расчет пройденной дистанции за 24 часа
Race - гонка.
*Поля - ArrayList с машинами.
Методы - добавление нового автомобиля в список. возврат победителя гонки
Main - Главный класс проекта.
Не получилось установить русскую локаль, в Main закомментированы тщетные попытки. Поэтому общение с пользователем на английском

Commit:
Remove classes

Commit:
Change project name

LionAlar committed:
Созданы 3 класса:
Car - автомобиль.
  * Поля - название и скорость
  * Методы - геттеры и сеттеры полей, проверка скорости на корректность и расчет пройденной дистанции за 24 часа
Race - гонка.
  *Поля - ArrayList с машинами.
  * Методы - добавление нового автомобиля в список. возврат победителя гонки
Main - Главный класс проекта.

LionAlar committed:
Merge branch 'main' into master

AleksandrIlinskii commented:
По поводу кодировки кирилицы в терминале - мне помогло в твоем проекте сделать вот что 

В меню «Help menu» -> «Edit Custom VM Options» добавьте в конец следующие параметры:

-Dconsole.encoding=UTF-8;
-Dfile.encoding=UTF-8.

И перезагрузить студию

AleksandrIlinskii commented:
![image](https://github.com/LionAlar/Java-Module-Project/assets/129396632/2ac5f255-5122-4afc-8e04-94a8ca237df1)

AleksandrIlinskii commented:
Приложение крашится, если ввести вместо чисел буквы 
![image](https://github.com/LionAlar/Java-Module-Project/assets/129396632/cdbe3973-bad2-4b7a-8e00-01df0af27e9c)

AleksandrIlinskii commented on src/main/java/Main.java:
```diff
@@ -1,6 +1,54 @@
+        String name;
+        double speed;
+        Car car;
+        for(int i = 0;i < 3;i++) {
```
Немного некорректное форматирование, нужно в каждом файле проекта выполнить меню code - reformat code, чтобы он привел все отступы под стандарт языка

AleksandrIlinskii commented on src/main/java/Race.java:
```diff
@@ -0,0 +1,23 @@
+    }
+
+    public Car getWinner(){
+        if (cars.size() == 0) return null;
```
Нужна лии тебе эта проверка?

LionAlar committed:
Созданы 3 класса:
Car - автомобиль.
  * Поля - название и скорость
  * Методы - геттеры и сеттеры полей, проверка скорости на корректность и расчет пройденной дистанции за 24 часа
Race - гонка.
  *Поля - ArrayList с машинами.
  * Методы - добавление нового автомобиля в список. возврат победителя гонки
Main - Главный класс проекта.

LionAlar committed:
Merge remote-tracking branch 'origin/master'

LionAlar committed:
Созданы 3 класса:
Car - автомобиль.
  * Поля - название и скорость
  * Методы - геттеры и сеттеры полей, проверка скорости на корректность и расчет пройденной дистанции за 24 часа
Race - гонка.
  *Поля - ArrayList с машинами.
  * Методы - добавление нового автомобиля в список. возврат победителя гонки
Main - Главный класс проекта.
